include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -32,20 +32,22 @@ ORT_DEFINE_DELETER(TensorTypeAndShapeInfo);
 namespace Ort {
 
 struct Exception : std::exception {
-  Exception(const char* message, OrtErrorCode code) : std::exception(message), code_{code} {}
+  Exception(std::string&& string, OrtErrorCode code) : message_{std::move(string)}, code_{code} {}
```

Comment:
Exception(std::string string,...) is enough?

Reply:
But then it'll be making a copy of the string every exception vs moving the existing one created in the macro?

Reply:
No. Compiler will optimize it.